[2023-06-06T00:49:05.945+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: process_weather_people_postgres.group_users.is_1_api_available scheduled__2023-06-05T00:00:00+00:00 [queued]>
[2023-06-06T00:49:05.956+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: process_weather_people_postgres.group_users.is_1_api_available scheduled__2023-06-05T00:00:00+00:00 [queued]>
[2023-06-06T00:49:05.957+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-06T00:49:05.957+0000] {taskinstance.py:1363} INFO - Starting attempt 7 of 10
[2023-06-06T00:49:05.957+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-06T00:49:05.972+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): group_users.is_1_api_available> on 2023-06-05 00:00:00+00:00
[2023-06-06T00:49:05.981+0000] {standard_task_runner.py:55} INFO - Started process 15926 to run task
[2023-06-06T00:49:05.984+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'process_weather_people_postgres', 'group_users.is_1_api_available', 'scheduled__2023-06-05T00:00:00+00:00', '--job-id', '205', '--raw', '--subdir', 'DAGS_FOLDER/process_weather_people_postgres/process_weather_people_postgres.py', '--cfg-path', '/tmp/tmponz79039']
[2023-06-06T00:49:05.986+0000] {standard_task_runner.py:83} INFO - Job 205: Subtask group_users.is_1_api_available
[2023-06-06T00:49:06.064+0000] {task_command.py:376} INFO - Running <TaskInstance: process_weather_people_postgres.group_users.is_1_api_available scheduled__2023-06-05T00:00:00+00:00 [running]> on host 1160da0dacdb
[2023-06-06T00:49:06.136+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=data_eng
AIRFLOW_CTX_DAG_ID=process_weather_people_postgres
AIRFLOW_CTX_TASK_ID=group_users.is_1_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-06-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=7
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-05T00:00:00+00:00
[2023-06-06T00:49:06.137+0000] {http.py:120} INFO - Poking: api/
[2023-06-06T00:49:06.148+0000] {base.py:71} INFO - Using connection ID 'random_people_api' for task execution.
[2023-06-06T00:49:06.150+0000] {http.py:148} INFO - Sending 'GET' to url: http://randomuser.me/api/
[2023-06-06T00:49:06.546+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-06-06T00:49:06.558+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=process_weather_people_postgres, task_id=group_users.is_1_api_available, execution_date=20230605T000000, start_date=20230606T004905, end_date=20230606T004906
[2023-06-06T00:49:06.599+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-06-06T00:49:06.639+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
